packages/fuels-abigen-macro/tests/test_projects/.gitignore

```diff
@@ -1,4 +0,0 @@
-**/out
-**/target
-**/Forc.lock
-**/.gitignore
```

Comment:
I don't think this file should be deleted -- isn't it what prevents us from accidentally committing auto-generated files from `forc build`?